src/VirtualClient/VirtualClient.UnitTests/WrathmarkTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace VirtualClient.UnitTests
+{
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using NUnit.Framework;
+    using VirtualClient.Actions.Wrathmark;
+    using VirtualClient.Contracts;
+
+    [TestFixture]
+    [Category("Unit")]
+    public class WrathmarkResultParserTests
```

Comment:
Thank you for adding the test, please move them to VirtualClient.Actions.UnitTests project